libraries/AP_AHRS/AP_AHRS_NavEKF.h

```diff
@@ -331,22 +331,16 @@ class AP_AHRS_NavEKF : public AP_AHRS_DCM {
     NavEKF3 EKF3;
 #endif
 
-private:
     enum class EKFType {
-        NONE = 0
-#if HAL_NAVEKF3_AVAILABLE
-        ,THREE = 3
-#endif
-#if HAL_NAVEKF2_AVAILABLE
-        ,TWO = 2
-#endif
-#if CONFIG_HAL_BOARD == HAL_BOARD_SITL
-        ,SITL = 10
-#endif
-#if HAL_EXTERNAL_AHRS_ENABLED
-        ,EXTERNAL = 11
-#endif
```

Comment:
This is what made you need to add all of those defaults in!  Why would you do this?!